uber/site_sections/preregistration.py

```diff
@@ -67,7 +67,11 @@ def unpaid_preregs(self):
     def paid_preregs(self):
         return cherrypy.session.setdefault('paid_preregs', [])
 
```

Comment:
Why this change?  The main reason to pass ``None`` and then instantiate a thing is if the argument is mutable.  For example, it's generally bad form to say ``def foo(bar=[])`` because ``[]`` is mutable and so the default value could theoretically change between invocations.  That's not really the case here (though technically I suppose someone could mutate the object if they went out of their way to do so).  I don't have any particular objection to it (though it does make the code slightly more complicated), but what was the reason for this change?